src/app/ui-library/button.tsx

```diff
@@ -6,10 +6,13 @@ type ButtonVariant = 'primary' | 'secondary' | 'tertiary' | 'danger';
 
 const StyledButton = styled.button`
   width: 100%;
+  min-height: 42.5px;
```

Comment:
question: is this something that is needed for all buttons and their variations? in zeroheight it's height: 44px.

if it's only for an edge case usage, consider applying this min-height where you need it rather than enforcing it here